recipe/meta.yaml

```diff
@@ -13,6 +14,9 @@ source:
 
 build:
   number: 100
```

Comment:
Would have to ask @pelson, but I go the impression that it was highly desirable to have this out by today.